akka-actor-typed-tests/src/test/scala/akka/actor/typed/LocalActorRefProviderLogMessagesSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed
+
+import akka.actor.testkit.typed.scaladsl.{ LoggingTestKit, ScalaTestWithActorTestKit }
+import akka.actor.typed.internal.adapter.ActorSystemAdapter
+import org.scalatest.wordspec.AnyWordSpecLike
+
+object LocalActorRefProviderLogMessagesSpec {
+  val config = """
+    akka {
+      loglevel = DEBUG # test verifies debug
+      log-dead-letters = on
+      actor {
+        debug.unhandled = on
+      }
+    }
+  """
+}
+
+class LocalActorRefProviderLogMessagesSpec
+    extends ScalaTestWithActorTestKit(LocalActorRefProviderLogMessagesSpec.config)
+    with AnyWordSpecLike {
```

Comment:
should also have `with LogCapturing`

Missing in several existing tests, so I added separate issue https://github.com/akka/akka/issues/29029